test/array.cpp

```diff
@@ -399,12 +399,16 @@ CAMP_TEST_BEGIN(array, structured_binding)
           a[1] == 3;
 } CAMP_TEST_END(array, structured_binding)
 
+#if defined(CAMP_ENABLE_CUDA) && (CAMP_CXX_STD == 17)
+//If CUDA is enabled and building with c++17, skip test to avoid NVCC error
+#else
 CAMP_TEST_BEGIN(array, deduction_guide)
 {
-   camp::array<int, 2> a{-1, 1};
+   camp::array a{-1, 1};
```

Comment:
Did you try Tom's suggestion of `camp::array a(-1, 1);`?

Reply:
Yes, when I do that I get the error
```
/g/g0/belcher6/camp/test/array.cpp(407): error: too many initializer values
/g/g0/belcher6/camp/test/array.cpp(407): error: no suitable constructor exists to convert from "int" to "camp::array<int, 2UL>"
```

Reply:
OK. I guess we can disable this test if CUDA is enabled.

Reply:
@adayton1, I did some digging around and there was a suggestion to update the cuda version. I updated to CUDA 11.8.0 and used clang/18.1.8 (loaded the clang/18.1.8-cuda-11.8.0-gcc-11.2.1 module) and it worked on rzansel. So - it looks like this is a bug which has been fixed on later cuda versions than I initially tried (I was running on 11.2.0 before).

Reply:
I could add a test for this in the azure pipelines... but I noticed that cuda isn't being turned on for the nvcc docker runs. is that done on purpose?

Reply:
Excellent.  I'm glad the update worked.  The extra error you posted means it didn't generate the right aggregate initializer like it should have.  Has the min CUDA version for RAJA advanced that far yet? If so, we might be able to get some compile time back on tuples for nvcc builds as well.

> I noticed that cuda isn't being turned on for the nvcc docker runs. is that done on purpose?

No, unless I missed it, and that's horrifying.  Creating an issue about that right now...